numpy/core/include/numpy/npy_1_7_deprecated_api.h

```diff
@@ -9,8 +9,8 @@
 #define _WARN___STR2__(x) #x
 #define _WARN___STR1__(x) _WARN___STR2__(x)
 #define _WARN___LOC__ __FILE__ "(" _WARN___STR1__(__LINE__) ") : Warning Msg: "
-#pragma message(_WARN___LOC__"Using deprecated NumPy API, disable it by " \
-                         "#defining NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION")
+#pragma message(_WARN___LOC__"Using deprecated NumPy API, disable it with " \
+                         "#define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION")
 #elif defined(__GNUC__)
 #warning "Using deprecated NumPy API, disable it by " \
          "#defining NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION"
```

Comment:
Missed the message duplicate on this line

Reply:
I'm not sure what do you mean by "message duplicate". Anyway I found two more lines which used `#defining` instead of `#define` and fixed them the same way.